code/index.html

```diff
@@ -1,22 +1,89 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="./style.css" />
+    <title>GAME ZONE NEWS</title>
   </head>
+
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+
+    <header>  <!--HEADER SECTION-->
+
+      <nav>
+        <img class="logo" src="./img/gamelogo.png" alt="GAME ZONE ICON">
+        <div class="left">
+          <a class="leftlinks" href="#">HOME</a>
+          <a class="leftlinks" href="#">GUIDE</a>
+        </div>
+        <div class="right">
+          <a class="rightlinks" href="#">UPCOMMING</a>
+          <a class="rightlinks" href="#">UPDATES</a>
+        </div>
+      </nav>
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
+    <main class="bigger-news-container"> <!-- HEADLINE NEWS SECTION-->
+      <section class="big-news-container">
+        <img src="./img/psxbox.png" alt="PS5 XBOX Image">
+
+        <div class="big-news-text">
+          <article>
+            <h2>Playstation 5 v/s Xbox Series X !</h2>
+            <p>The next generation of consoles is coming, and Sony and Microsoft are getting ready to battle it out for system supremacy, pitting the PlayStation 5 against the Xbox Series X this holiday season.</p>
+          </article>
+          <a class="readmore" ref="https://www.tomsguide.com/face-off/ps5-vs-xbox-series-x">Read More...</a>
+        </div>
+      </section>
+    </main>
+
+    <hr/>
+
+    <h3 class="latestnews.">GAMES YOU SHOULD PLAY ...</h3>
+
+    <section class="newscards">  <!-- NEWS CARDS-->
+     
+      <div class="cards">
```

Comment:
Think about creating clear class names for your elements. This is a singular card